2017-06-02 15:38:15 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Categories could not be found.
Exception Attributes: array (
  'class' => 'Categories',
  'plugin' => false,
  'prefix' => false,
  '_ext' => false,
)
Request URL: /categories
Referer URL: http://localhost/cakeblog/blog/articles
Stack Trace:
#0 C:\xampp\htdocs\cakeblog\blog\vendor\cakephp\cakephp\src\Http\ControllerFactory.php(70): Cake\Http\ControllerFactory->missingController(Object(Cake\Http\ServerRequest))
#1 C:\xampp\htdocs\cakeblog\blog\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(90): Cake\Http\ControllerFactory->create(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response))
#2 C:\xampp\htdocs\cakeblog\blog\vendor\cakephp\cakephp\src\Http\BaseApplication.php(78): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response))
#3 C:\xampp\htdocs\cakeblog\blog\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response), Object(Cake\Http\Runner))
#4 C:\xampp\htdocs\cakeblog\blog\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(59): Cake\Http\Runner->__invoke(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response))
#5 C:\xampp\htdocs\cakeblog\blog\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response), Object(Cake\Http\Runner))
#6 C:\xampp\htdocs\cakeblog\blog\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response))
#7 C:\xampp\htdocs\cakeblog\blog\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response), Object(Cake\Http\Runner))
#8 C:\xampp\htdocs\cakeblog\blog\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(92): Cake\Http\Runner->__invoke(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response))
#9 C:\xampp\htdocs\cakeblog\blog\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response), Object(Cake\Http\Runner))
#10 C:\xampp\htdocs\cakeblog\blog\vendor\cakephp\debug_kit\src\Middleware\DebugKitMiddleware.php(52): Cake\Http\Runner->__invoke(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response))
#11 C:\xampp\htdocs\cakeblog\blog\vendor\cakephp\cakephp\src\Http\Runner.php(65): DebugKit\Middleware\DebugKitMiddleware->__invoke(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response), Object(Cake\Http\Runner))
#12 C:\xampp\htdocs\cakeblog\blog\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response))
#13 C:\xampp\htdocs\cakeblog\blog\vendor\cakephp\cakephp\src\Http\Server.php(80): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Cake\Http\ServerRequest), Object(Cake\Http\Response))
#14 C:\xampp\htdocs\cakeblog\blog\webroot\index.php(37): Cake\Http\Server->run()
#15 {main}


2017-06-02 16:05:05 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Categories could not be found.
Exception Attributes: array (
  'class' => 'Categories',
  'plugin' => false,
  'prefix' => false,
  '_ext' => false,
)
Request URL: /categories
Stack Trace:
#0 C:\xampp\htdocs\cakeblog\blog\vendor\cakephp\cakephp\src\Http\ControllerFactory.php(70): Cake\Http\ControllerFactory->missingController(Object(Cake\Http\ServerRequest))
#1 C:\xampp\htdocs\cakeblog\blog\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(90): Cake\Http\ControllerFactory->create(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response))
#2 C:\xampp\htdocs\cakeblog\blog\vendor\cakephp\cakephp\src\Http\BaseApplication.php(78): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response))
#3 C:\xampp\htdocs\cakeblog\blog\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response), Object(Cake\Http\Runner))
#4 C:\xampp\htdocs\cakeblog\blog\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(59): Cake\Http\Runner->__invoke(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response))
#5 C:\xampp\htdocs\cakeblog\blog\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response), Object(Cake\Http\Runner))
#6 C:\xampp\htdocs\cakeblog\blog\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response))
#7 C:\xampp\htdocs\cakeblog\blog\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response), Object(Cake\Http\Runner))
#8 C:\xampp\htdocs\cakeblog\blog\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(92): Cake\Http\Runner->__invoke(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response))
#9 C:\xampp\htdocs\cakeblog\blog\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response), Object(Cake\Http\Runner))
#10 C:\xampp\htdocs\cakeblog\blog\vendor\cakephp\debug_kit\src\Middleware\DebugKitMiddleware.php(52): Cake\Http\Runner->__invoke(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response))
#11 C:\xampp\htdocs\cakeblog\blog\vendor\cakephp\cakephp\src\Http\Runner.php(65): DebugKit\Middleware\DebugKitMiddleware->__invoke(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response), Object(Cake\Http\Runner))
#12 C:\xampp\htdocs\cakeblog\blog\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response))
#13 C:\xampp\htdocs\cakeblog\blog\vendor\cakephp\cakephp\src\Http\Server.php(80): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Cake\Http\ServerRequest), Object(Cake\Http\Response))
#14 C:\xampp\htdocs\cakeblog\blog\webroot\index.php(37): Cake\Http\Server->run()
#15 {main}


2017-06-02 18:20:33 Error: [ParseError] syntax error, unexpected ';', expecting ',' or ')' in C:\xampp\htdocs\cakeblog\blog\src\Controller\CategoriesController.php on line 26
Request URL: /categories
Referer URL: http://localhost/cakeblog/blog/
Stack Trace:
#0 C:\xampp\htdocs\cakeblog\blog\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('C:\\xampp\\htdocs...')
#1 [internal function]: Composer\Autoload\ClassLoader->loadClass('App\\Controller\\...')
#2 [internal function]: spl_autoload_call('App\\Controller\\...')
#3 C:\xampp\htdocs\cakeblog\blog\vendor\cakephp\cakephp\src\Core\App.php(152): class_exists('App\\Controller\\...')
#4 C:\xampp\htdocs\cakeblog\blog\vendor\cakephp\cakephp\src\Core\App.php(64): Cake\Core\App::_classExistsInBase('\\Controller\\Cat...', 'App')
#5 C:\xampp\htdocs\cakeblog\blog\vendor\cakephp\cakephp\src\Http\ControllerFactory.php(68): Cake\Core\App::className('Categories', 'Controller', 'Controller')
#6 C:\xampp\htdocs\cakeblog\blog\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(90): Cake\Http\ControllerFactory->create(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response))
#7 C:\xampp\htdocs\cakeblog\blog\vendor\cakephp\cakephp\src\Http\BaseApplication.php(78): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response))
#8 C:\xampp\htdocs\cakeblog\blog\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response), Object(Cake\Http\Runner))
#9 C:\xampp\htdocs\cakeblog\blog\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(59): Cake\Http\Runner->__invoke(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response))
#10 C:\xampp\htdocs\cakeblog\blog\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response), Object(Cake\Http\Runner))
#11 C:\xampp\htdocs\cakeblog\blog\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response))
#12 C:\xampp\htdocs\cakeblog\blog\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response), Object(Cake\Http\Runner))
#13 C:\xampp\htdocs\cakeblog\blog\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(92): Cake\Http\Runner->__invoke(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response))
#14 C:\xampp\htdocs\cakeblog\blog\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response), Object(Cake\Http\Runner))
#15 C:\xampp\htdocs\cakeblog\blog\vendor\cakephp\debug_kit\src\Middleware\DebugKitMiddleware.php(52): Cake\Http\Runner->__invoke(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response))
#16 C:\xampp\htdocs\cakeblog\blog\vendor\cakephp\cakephp\src\Http\Runner.php(65): DebugKit\Middleware\DebugKitMiddleware->__invoke(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response), Object(Cake\Http\Runner))
#17 C:\xampp\htdocs\cakeblog\blog\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Cake\Http\ServerRequest), Object(Cake\Http\Response))
#18 C:\xampp\htdocs\cakeblog\blog\vendor\cakephp\cakephp\src\Http\Server.php(80): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Cake\Http\ServerRequest), Object(Cake\Http\Response))
#19 C:\xampp\htdocs\cakeblog\blog\webroot\index.php(37): Cake\Http\Server->run()
#20 {main}


2017-06-05 08:56:12 Warning: Warning (2): session_destroy(): Session object destruction failed in [C:\xampp\htdocs\cakeblog\blog\vendor\cakephp\cakephp\src\Network\Session.php, line 509]
Request URL: /
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 153
session_destroy - [internal], line ??
Cake\Network\Session::destroy() - CORE\src\Network\Session.php, line 509
Cake\Network\Session::start() - CORE\src\Network\Session.php, line 333
Cake\Network\Session::check() - CORE\src\Network\Session.php, line 360
Cake\View\Helper\FlashHelper::render() - CORE\src\View\Helper\FlashHelper.php, line 73
include - APP/Template\Layout\default.ctp, line 50
Cake\View\View::_evaluate() - CORE\src\View\View.php, line 1011
Cake\View\View::_render() - CORE\src\View\View.php, line 972
Cake\View\View::renderLayout() - CORE\src\View\View.php, line 643
Cake\View\View::render() - CORE\src\View\View.php, line 601
Cake\Controller\Controller::render() - CORE\src\Controller\Controller.php, line 623
Cake\Http\ActionDispatcher::_invoke() - CORE\src\Http\ActionDispatcher.php, line 125
Cake\Http\ActionDispatcher::dispatch() - CORE\src\Http\ActionDispatcher.php, line 93
Cake\Http\BaseApplication::__invoke() - CORE\src\Http\BaseApplication.php, line 78
Cake\Http\Runner::__invoke() - CORE\src\Http\Runner.php, line 65
Cake\Routing\Middleware\RoutingMiddleware::__invoke() - CORE\src\Routing\Middleware\RoutingMiddleware.php, line 59
Cake\Http\Runner::__invoke() - CORE\src\Http\Runner.php, line 65
Cake\Routing\Middleware\AssetMiddleware::__invoke() - CORE\src\Routing\Middleware\AssetMiddleware.php, line 88
Cake\Http\Runner::__invoke() - CORE\src\Http\Runner.php, line 65
Cake\Error\Middleware\ErrorHandlerMiddleware::__invoke() - CORE\src\Error\Middleware\ErrorHandlerMiddleware.php, line 92
Cake\Http\Runner::__invoke() - CORE\src\Http\Runner.php, line 65
DebugKit\Middleware\DebugKitMiddleware::__invoke() - ROOT\vendor\cakephp\debug_kit\src\Middleware\DebugKitMiddleware.php, line 52
Cake\Http\Runner::__invoke() - CORE\src\Http\Runner.php, line 65
Cake\Http\Runner::run() - CORE\src\Http\Runner.php, line 51
Cake\Http\Server::run() - CORE\src\Http\Server.php, line 80
[main] - ROOT\webroot\index.php, line 37



